jep/1/README.adoc

```diff
@@ -546,6 +523,21 @@ If one or more contributors are not responding
 and the sponsor chooses to move forard without their feedback,
 they should document that choice in the "<<Reasoning>>" section of the JEP.
 
+==== Resolving Disputes
+
+Except for the decision by a reviewer to accept or reject a JEP,
+the JEP process is run by
+link:https://en.wikipedia.org/wiki/Consensus_decision-making[consensus].
+It is the responsibility of the every contributor to respect other contributors,
+listen to their perspectives, and attempt to find solution that work for everyone.
+If consensus cannot be achieved or if there is strong disagreement with the decision of a reviewer,
+a contributor may request that the Jenkins Governance Board intervene.
+The board will take up the matter at their next regular meeting and render a decision at the meeting following.
+
+This should be a last resort and should happen extremely rarely.
+
```

Comment:
"Governance meeting may make a decision to return JEP to draft and to assign another reviewer".
I am not sure whether Governance Meeting should be a place for "accepting"/"rejecting" JEPs though formally it should be eligible to do that as an ultimate power entity in Jenkins org.

Reply:
I considered adding some statement that said JEPs could be rejected or returned to draft via this path, but not accepted.  Only reviewers can accept, but as I said below, please go write it up as you believe it should be.